src/sentry/static/sentry/app/views/projectInstall/platform.jsx

```diff
@@ -182,10 +198,50 @@ const ProjectInstallPlatform = createReactClass({
     );
   },
 
+  renderTestBody() {
+    let {integration, platform} = this.state;
+    let {dsnPublic} = this.props.platformData;
+
+    if (!integration || !platform) {
+      return <NotFound />;
+    }
+
+    analytics('experiment.installation_instructions', {
```

Comment:
does this have side effects? it might be better not to have it in the render method